tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -29,17 +28,18 @@ def test_selinux_status(target_sat):
 
     :id: 43218070-ac5e-4679-b74a-3e2bcb497a0a
 
-    :expectedresults: SELinux is enabled and there are no denials
+    :expectedresults: SELinux is enabled and there are no denials for pulp services
 
-    :BZ: 2263294
+    :Verifies: SAT-23121
     """
     # check SELinux is enabled
     result = target_sat.execute('getenforce')
     assert 'Enforcing' in result.stdout
     # check there are no SELinux denials
-    if not is_open('SAT-23121'):
-        result = target_sat.execute('ausearch --input-logs -m avc -ts today --raw')
-        assert result.status == 1, 'Some SELinux denials were found in journal.'
+    result = target_sat.execute(
+        'ausearch --input-logs -m avc -ts today --raw | grep \'comm="pulp\''
+    )
+    assert result.status == 1, 'Some SELinux denials were found in journal.'
```

Comment:
`grep` returns 1 if there were no matches, and 0 if there were, so this will assert the opposite of what was intended ;)